.codecov.yml

```diff
@@ -1,29 +1,65 @@
-comment: off
-ignore:
-  # Auto generated
-  - "^.*_gen.go$"
-  - "^.*/mock_full.go$"
-  - "^chain/actors/builtin/[^/]*/(message|state|v)[0-4]\\.go$" # We test the latest version only.
-  # Tests
-  - "api/test/**"
-  - "conformance/**"
-  # Generators
-  - "gen/**"
-  - "chain/actors/agen/**"
-  # Non-critical utilities
-  - "cmd/lotus-shed/**"
-  - "cmd/tvx/**"
-  - "cmd/lotus-pcr/**"
-  - "cmd/tvx/**"
-  - "cmd/lotus-health/**"
-  - "cmd/lotus-fountain/**"
-  - "cmd/lotus-stats/**"
-  - "cmd/lotus-pcr/**"
-  - "cmd/lotus-sim/**"
-  - "cmd/lotus-bench/**"
-  - "cmd/lotus-seed/**"
-  - "cmd/chain-noise/**"
-  - "api/docgen/**"
-  - "api/docgen-openrpc/**"
-github_checks:
-  annotations: false
+coverage:
+  status:
+    project:
+      tools-and-tests:
+        target: auto
+        threshold: 0.1%
+        informational: true
+        paths:
+          - "testplans"
+          - "tools"
+          - "system"
+          - "snap"
+          - "lotuspond"
+          - "conformance"
+          - "scripts"
+          - "gen"
+          - "build"
+      markets: 
+        target: auto
+        threshold: 0.1%
+        informational: true        
+        paths: 
+          - "markets"
+          - "paychmgr"
+      miner: 
+        target: auto
+        threshold: 0.1%
+        informational: true        
+        paths: 
+          - "miner"
+          - "storage"
+      chain:
+        target: auto
+        threshold: 0.1%
+        informational: true        
```

Comment:
Probably sufficient to just set this to `false` -- we could subdivide into smaller projects, but I think they'd all still be "required" instead of "informational"